src/app/ReadClient.h

```diff
@@ -84,11 +103,17 @@ class ReadClient : public Messaging::ExchangeDelegate
 
     enum class ClientState
     {
-        Uninitialized = 0,     ///< The client has not been initialized
-        Initialized,           ///< The client has been initialized and is ready for a SendReadRequest
-        AwaitingInitialReport, ///< The client is waiting for initial report
+        Uninitialized = 0,         ///< The client has not been initialized
+        Initialized,               ///< The client has been initialized and is ready for a SendReadRequest
+        AwaitingInitialReport,     ///< The client is waiting for initial report
+        AwaitingSubscribeResponse, ///< The client is waiting for subscribe response
+        SubscriptionIdle,          ///< The client is maintaining subscription
```

Comment:
Unless Idle comes from the spec, I suggest perhaps renaming this.  Idle to me implies we aren't doing anything.  But that's not true.  This is the active state for the subscription that we arrive at after the successful handshake.  In fact, `SubscriptionActive` might be a better name.

Reply:
Done, thanks

Reply:
How about `SubscriptionAlive`, since `Active` maybe intepreted as "we are sending reports now"